doc/windows/build_instructions.md

```diff
@@ -176,3 +176,17 @@ cd build
 cmake -DCMAKE_PREFIX_PATH="C:\projects\aditof-sdk\deps\glog\build_0_3_5\local_path\glog;C:\projects\aditof-sdk\deps\protobuf\build_3_9_0\local_path\protobuf;C:\projects\aditof-sdk\deps\libwebsockets\build_3_2_3\local_path\websockets;C:\projects\aditof-sdk\deps\opencv\build_4_5_0\local_path\opencv" -G "Visual Studio 16 2019" -DOPENSSL_INCLUDE_DIRS="C:\OpenSSL-Win64\include" -DWITH_EXAMPLES=on ..
 cmake --build . --target install --config Release -j 4
 ```
+
+## Build doxygen documentation
+
+Requirements:
+* Doxygen
+* Graphviz
+> NOTE: You might need to edit the PATH environment variable and reboot your system.
```

Comment:
I didn't have to do any of these. Just to close and restart my command prompt window.